.drone.yml

```diff
@@ -89,84 +92,84 @@ steps:
     name: integration-vsphere
     commands:
       - bats -t ./automated-tests/integration/vsphere/tests.sh
-# TODO fix e2e tests
-#  - &e2e
-#    name: e2e-gcp
-#    image: quay.io/sighup/e2e-furyctl:v1.2.1_v0.2.2_v1.20.1_20.04
-#    pull: always
-#    privileged: true # Required to connect to the VPN
-#    depends_on:
-#      - build
-#    environment:
-#      FURYCTL_TOKEN:
-#        from_secret: FURYCTL_TOKEN
-#
-#      TERRAFORM_TF_STATES_BUCKET_NAME:
-#        from_secret: TERRAFORM_TF_STATES_BUCKET_NAME
-#      AWS_ACCESS_KEY_ID:
-#        from_secret: AWS_ACCESS_KEY_ID
-#      AWS_SECRET_ACCESS_KEY:
-#        from_secret: AWS_SECRET_ACCESS_KEY
-#      AWS_REGION:
-#        from_secret: AWS_REGION
-#      AWS_DEFAULT_REGION:
-#        from_secret: AWS_REGION
-#
-#      GCP_CREDENTIALS:
-#        from_secret: GCP_CREDENTIALS
-#      GCP_CREDENTIALS_PATH: /tmp/terraform-credentials.json
-#
-#      GOOGLE_PROJECT:
-#        from_secret: GOOGLE_PROJECT
-#      CLOUDSDK_CORE_PROJECT:
-#        from_secret: GOOGLE_PROJECT
-#      GOOGLE_REGION:
-#        from_secret: GOOGLE_REGION
-#
-#      VSPHERE_USER:
-#        from_secret: VSPHERE_USER
-#      VSPHERE_PASSWORD:
-#        from_secret: VSPHERE_PASSWORD
-#      VSPHERE_SERVER:
-#        from_secret: VSPHERE_SERVER
-#      VSPHERE_DATACENTER:
-#        from_secret: VSPHERE_DATACENTER
-#      VSPHERE_DATASTORE:
-#        from_secret: VSPHERE_DATASTORE
-#      VSPHERE_HOST:
-#        from_secret: VSPHERE_HOST
-#      VSPHERE_NET:
-#        from_secret: VSPHERE_NET
-#      VSPHERE_TEMPLATE_PREFIX:
-#        from_secret: VSPHERE_TEMPLATE_PREFIX
-#    commands:
-#      - echo $${GCP_CREDENTIALS} | base64 -d > $${GCP_CREDENTIALS_PATH}
-#      - export GOOGLE_APPLICATION_CREDENTIALS=$${GCP_CREDENTIALS_PATH}
-#      - bats -t ./automated-tests/e2e-tests/gcp-gke/tests.sh
-#    when:
-#      event:
-#        - tag
-#
-#  - <<: *e2e
-#    name: e2e-aws
-#    commands:
-#      - bats -t ./automated-tests/e2e-tests/aws-eks/tests.sh
-#
-#  - <<: *e2e
-#    name: e2e-vsphere
-#    commands:
-#      - bats -t ./automated-tests/e2e-tests/vsphere/tests.sh
-#
-#  - <<: *e2e
-#    name: e2e-vsphere-destroy
-#    depends_on:
-#      - e2e-vsphere
-#    failure: ignore
-#    commands:
-#      - echo " Our vsphere environment is not so stable"
-#      - echo " The destroy phase fails randomly because of disconnection events between vsphere nodes and the API"
-#      - echo " Ignore errors on this step"
-#      - bats -t ./automated-tests/e2e-tests/vsphere/tests-destroy.sh
+
+  - &e2e
+    name: e2e-gcp
+    image: quay.io/sighup/e2e-furyctl:v1.2.1_v0.2.2_v1.20.1_20.04
+    pull: always
+    privileged: true # Required to connect to the VPN
```

Comment:
If the pipeline runs on drone master, we don't need this anymore
```suggestion

```
This was needed when we had drone hosted on the cloud instead of our vSphere env

Reply:
We need the parameter for the VPN connection on EKS and GKE environments